pysph/sph/acceleration_eval_opencl_helper.py

```diff
@@ -81,9 +82,15 @@ def _call_kernel(self, info, extra_args):
                 cache._start_idx_gpu.array.data,
                 cache._neighbors_gpu.array.data
             ] + extra_args
-            call(*args)
+            event = call(*args)
+            if self._profile:
+                profile(call.get_info(cl.kernel_info.FUNCTION_NAME),
```

Comment:
I think `call.function_name` will also work just fine instead of this additional `call.get_info` method.